source/triggers/database-triggers.txt

```diff
@@ -635,9 +635,14 @@ Expression` field. App Services evaluates the match expression against the
 change event document and invokes the Trigger only if the expression evaluates
 to true for the given change event.
 
+The "match expression" is basically a JSON document that specifies the query
+conditions using the :manual:`MongoDB read query syntax
+</reference/operator/query/>`.
+
 We recommend only using match expressions when the volume of Trigger events
```

Comment:
We have "match expression" in quotes above and not here. I think it's harder to scan here without the quotes.

Reply:
I will remove quotation marks, this is the only place I used them (for my own reasons)